pythonforandroid/recipes/cryptography/__init__.py

```diff
@@ -4,9 +4,9 @@
 
 class CryptographyRecipe(CompiledComponentsPythonRecipe):
     name = 'cryptography'
-    version = '1.3'
-    url = 'https://github.com/pyca/cryptography/archive/{version}.tar.gz'
-    depends = [('python2', 'python3crystax'), 'openssl', 'idna', 'pyasn1', 'six', 'setuptools', 'enum34', 'ipaddress', 'cffi']
+    version = '2.3.1'
+    url = 'https://files.pythonhosted.org/packages/22/21/233e38f74188db94e8451ef6385754a98f3cad9b59bedf3a8e8b14988be4/cryptography-2.3.1.tar.gz'
```

Comment:
Please don't change the `url` string and don't hardcode the version

Reply:
@AndreMiras previous URL is not restored